xray/plot/plot.py

```diff
@@ -414,13 +419,27 @@ def _plot2d(plotfunc):
     @functools.wraps(plotfunc)
     def newplotfunc(darray, ax=None, xincrease=None, yincrease=None,
                     add_colorbar=True, add_labels=True, vmin=None, vmax=None, cmap=None,
-                    center=None, robust=False, extend=None, levels=None,
+                    center=None, robust=False, extend=None, levels=None, colors=None,
```

Comment:
I think you still need to fix this. Also, I would put `colors` immediately following `cmap` in the function argument.